File: back/src/main/java/com/bank/account/dto/OperationRequestDTO.java
package com.bank.account.dto;

import com.bank.account.enumeration.TypeOperation;
import lombok.*;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
@Builder
@EqualsAndHashCode
public class OperationRequestDTO {
    Long idAccountBank;
    Long somme;
    TypeOperation typeOperation;
}

File: front/src/interface/operationRequestDTO.ts
import { TypeOperation } from "./typeOperation";

export class  OperationRequestDTO {
    public idAccountBank?: number;
    public somme?: number;
    public typeOperation?: TypeOperation;
    constructor (idAccountBank: number, somme: number ,typeOperation: TypeOperation){
        this.idAccountBank = idAccountBank;
        this.somme = somme;
        this.typeOperation = typeOperation;
    }
}